test/segments/dir.spec

```diff
@@ -72,6 +72,26 @@ function testTruncationFromRightWorks() {
   unset POWERLEVEL9K_SHORTEN_STRATEGY
 }
 
+function testTruncationFromRightWithEmptyDelimiter() {
+  POWERLEVEL9K_SHORTEN_DIR_LENGTH=2
```

Comment:
Use `local` for the `POWERLEVEL9K_*` variables and `FOLDER`; then you don't need `unset` at the end.